src/common/services/cortexApi.service.js

```diff
@@ -66,8 +65,8 @@ class CortexApi {
     return this.http(request);
   }
 
-  serializePath(path){
-    if(isArray(path)){
+  static serializePath( path){
```

Comment:
I yanked it back out to fix the specs. I originally did it because of the intellij suggestion.